db/migrate/20230804084334_add_trophies_mailer_communication_preference.rb

```diff
@@ -0,0 +1,7 @@
+class AddTrophiesMailerCommunicationPreference < ActiveRecord::Migration[7.0]
+  def change
+    return if Rails.env.production?
+
+    add_column :user_communication_preferences, :email_on_acquired_trophy_notification, :boolean, default: true, null: false
```

Comment:
I'd like to set this to the value of `email_on_acquired_badge_notification`, so:

```ruby
User::CommunicationPreferences.update_all('email_on_acquired_trophy_notification = email_on_acquired_badge_notification'
```